x-pack/legacy/plugins/ml/public/jobs/jobs_list/components/jobs_list_view/jobs_list_view.js

```diff
@@ -94,7 +95,7 @@ export class JobsListView extends Component {
   }
 
   componentWillUnmount() {
-    timefilter.off('refreshIntervalUpdate');
+    this.refreshIntervalSubscription.unsubscribe();
```

Comment:
This might need a check if `this.refreshIntervalSubscription` is not `null`?

Reply:
done